src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator, InfoBox {
```

Comment:
```suggestion
public class Circle extends Figure  {
```
якщо Figure імплементує ці інтерфейси, то і наслідники автоматично це робитимуть